### ISSUE TITLE
test_statusflags is failing on Travis

### ISSUE THREAD

**Issue by marco-c** [mozilla/clouseau#82]
```
FAIL: test_get (test_statusflags.StatusFlagTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "<string>", line 3, in wrapper
  File "/home/travis/build/mozilla/clouseau/tests/test_statusflags.py", line 186, in test_get
    self.assertEqual(isgn['rank']['release'], getnumbers(-1, 20, -1))
AssertionError: {'plugin': -1, 'content': 18, 'browser': -1} != {'plugin': -1, 'content': 20, 'browser': -1}
- {'browser': -1, 'content': 18, 'plugin': -1}
?                            ^^

+ {'browser': -1, 'content': 20, 'plugin': -1}
? 
```

**Comment by calixteman** [comment]
Yep I know. I need to fix a problem with final date.

**Comment by marco-c** [comment]
@calixteman news? We risk introducing new bugs because we don't notice new failures.
E.g. https://github.com/mozilla/clouseau/pull/94 introduced flake8 failures, https://github.com/mozilla/clouseau/pull/96 caused the build to fail on Python 3.

**Comment by calixteman** [comment]
I know... I need to take some time to fix issues in status_flags test

**Comment by marco-c** [comment]
I will file new issues for the two new problems.

**Comment by marco-c** [comment]
We also have a new status_flags test failure:

```
Traceback (most recent call last):
  File "<string>", line 3, in wrapper
  File "/home/travis/build/mozilla/clouseau/tests/test_statusflags.py", line 155, in test_get
    self.assertEqual(info['base_versions']['aurora'], 50)
AssertionError: 51 != 50
```

The best way to fix this is probably to introduce mocks like in libmozdata, so that the tests don't have to change when we release new versions.

**Comment by marco-c** [comment]
Fixed by #100.
